src/sentry/static/sentry/app/components/issues/compactIssue.jsx

```diff
@@ -281,3 +281,9 @@ const StyledErrorLevel = styled(ErrorLevel)`
   display: block;
   margin-right: ${space(1)};
 `;
+
+const ChatLink = styled(Link)`
+  & > svg {
+    margin-right: ${space(0.5)};
+  }
```

Comment:
Why did the old icon not need this margin?

Reply:
Because it was an icon font icon 😬